### PR TITLE
Use netcoreapp 1.0.5 shared runtime.

### PR DESCRIPTION
**Pull request by codito**
Fixes an issue in case-sensitive OS where packages are not located
with netcoreapp2.0 sdk and 1.0.4 runtime. See dotnet/cli#6390.

### CODE REVIEW

**Comment by codito** [comment]
This doesn't work out of box since `1.0.4` doesn't automatically roll forward to `1.0.5` servicing.

**Comment by codito** [comment]
Abandoning this for now, will bring back when 1.0.5 is shipped.

**Comment by codito** [comment]
@dotnet-bot test Windows_NT please

**Comment by codito** [comment]
@dotnet-bot test this please

**Review by harshjain2 — APPROVED** [review]
👍

File: src/Microsoft.TestPlatform.Common/DataCollection/DataCollectionAttachmentManager.cs [hunk 1]
```diff
@@ -53,7 +59,17 @@ internal class DataCollectionAttachmentManager : IDataCollectionAttachmentManage
         /// Initializes a new instance of the <see cref="DataCollectionAttachmentManager"/> class.
         /// </summary>
         public DataCollectionAttachmentManager()
+            : this(new Microsoft.VisualStudio.TestPlatform.Utilities.Helpers.FileHelper())
```
Comment (harshjain2) [inline src/Microsoft.TestPlatform.Common/DataCollection/DataCollectionAttachmentManager.cs]: nit : add `using Microsoft.VisualStudio.TestPlatform.Utilities.Helpers;`

File: src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs [hunk 2]
```diff
@@ -58,5 +58,17 @@ public FileAttributes GetFileAttributes(string path)
         {
             return new FileInfo(path).Attributes;
         }
+
+        /// <inheritdoc/>
+        public void CopyFile(string sourcePath, string destinationPath)
```
Comment (harshjain2) [inline src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs]: We don't have tests for `FileHelper`?

Comment (codito) [inline src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs]: Yes, it doesn't have any business logic.

File: src/Microsoft.TestPlatform.Common/DataCollection/DataCollectionAttachmentManager.cs (hunk 1 above)
Comment (codito) [inline src/Microsoft.TestPlatform.Common/DataCollection/DataCollectionAttachmentManager.cs]: I will fix this in a different PR. Have to unblock the PR/CI immediately.

**Pull request merged**